dsf-gdb/org.eclipse.cdt.dsf.gdb/src/org/eclipse/cdt/dsf/mi/service/command/CommandFactory.java

```diff
@@ -895,6 +896,11 @@ public ICommand<MIInfo> createMIGDBSetTargetAsync(ICommandControlDMContext ctx,
 		return new MIGDBSetTargetAsync(ctx, isSet);
```

Comment:
Should we mark createMIGDBSetTargetAsync() as deprecated?

Reply:
No, only because it wouldn't match how we have dealt with all the support for older GDB. If we start down that road a huge amount of DSF-GDB would get marked as deprecated, and I don't think there is appetite for that.